PDFParse/pdfsort.ts

```diff
@@ -0,0 +1,62 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+interface ParsedInfo {
+  category: string;
+  content: string[];
+}
+
+function isName(line: string, isFirstNonEmptyLine: boolean): string | null {
+  const trimmedLine = line.trim();
+  return isFirstNonEmptyLine && trimmedLine.length > 0 ? trimmedLine : null;
+}
+
+function parseResume(txtPath: string): ParsedInfo[] {
+  const lines = fs.readFileSync(txtPath, 'utf-8').split('\n');
+
+  const parsedInfo: ParsedInfo[] = [];
+  let currentCategory: string | null = null;
+  let isFirstNonEmptyLine = true;
+
+  lines.forEach(line => {
+    // Skip empty lines
+    if (!line.trim()) {
+      return;
+    }
+
+    // Check if the line is the name
+    const name = isName(line, isFirstNonEmptyLine);
+    if (name) {
+      parsedInfo.push({ category: 'Name', content: [name] });
+      isFirstNonEmptyLine = false; // Set to false after finding the first non-empty line
+      return;
+    }
```

Comment:
What is "name" supposed to be. At first glance I keep thinking this is the name of the person but then I think its  a position name or a section heading.